frameworks/digital-outcomes-and-specialists/questions/brief-responses/legacyNiceToHaveRequirements.yml

```diff
@@ -0,0 +1,21 @@
+name: Nice-to-have requirements
+id: niceToHaveRequirements
+question: Do you have any of the nice-to-have skills and experience?
+question_advice: |
+  Buyers will filter supplier applications based on the number of nice-to-have skills and experience they have.
+
+type: boolean_list
+number_of_items: 20
+optional: true
+
+depends:
```

Comment:
Is depending on all lots that have briefs the same as depending on no lots at all? It's nice to have it explicit I guess, but other brief response questions that apply to all lots don't have a "depends" section at all.